redaxo/src/addons/backup/lib/backup.php

```diff
@@ -329,6 +329,10 @@ public static function exportDb($filename, array $tables = null)
             }
         }
         foreach ($tables as $table) {
+            if (!preg_match('/^[a-z0-9._-]+$/i', $table)) {
```

Comment:
Prinzipiell ist vieles erlaubt in neuen versionen... wie genau will man werden...?